utils/erc721ForMetaplex.py

```diff
@@ -69,14 +70,16 @@ def safe_mint(self, seed, to_address, uri, data=None, gas_price=None, gas=None,
         return logs[0]["args"]["tokenId"]
 
     @allure.step("Transfer from")
-    def transfer_from(self, address_from, address_to, token_id, signer, gas_price=None, gas=None):
+    @stats_collector.cost_report
+    def transfer_from(self, address_from, address_to, token_id, signer, gas_price=None, gas=None) -> TxReceipt:
         tx = self.make_tx_object(signer.address, gas_price, gas)
         instruction_tx = self.contract.functions.transferFrom(address_from, address_to, token_id).build_transaction(tx)
         resp = self.web3_client.send_transaction(signer, instruction_tx)
         return resp
 
     @allure.step("Safe transfer from")
-    def safe_transfer_from(self, address_from, address_to, token_id, signer, data=None, gas_price=None, gas=None):
+    @stats_collector.cost_report
+    def safe_transfer_from(self, address_from, address_to, token_id, signer, data=None, gas_price=None, gas=None) -> TxReceipt:
```

Comment:
no needs to safe statistic for every transfer happens during the test run
Can we do it only once? 
to move @stats_collector.cost_report from utils to the  certain tests in TestERC721 and TestERC20SPL?

Reply:
here's how this works

- pytest ... --cost_report_dir=... - enables saving cost reports data
- @pytest.mark.cost_report - marks tests for which cost report data should be collected
- @stats_collector.cost_report_from_receipt - decorates any function that returns Ethereum-like receipt and should collect data from the receipt but it works only if 

  1.   --cost_report_dir=... was used
  2.   this function was called from a test decorated by @pytest.mark.cost_report